init_positions:
- -0.31527358293533325
- -1.6674665212631226
#- 1.8415489196777344
- 1.4415489196777344
#- -1.744444727897644
- -1.344444727897644
- -1.5690969228744507
- -0.31525498628616333
place_positions:
- -1.5650523900985718
- -1.6251858472824097
- 1.78648042678833
- -1.7326310873031616
- -1.5690850019454956
- 0.0014133895747363567
place_pre_positions:
- -1.5657953023910522
- -1.6183360815048218
- 1.4160847663879395
- -1.3690685033798218
- -1.56961190700531
- 0.0017721672775223851
